be/src/common/config.h

```diff
@@ -801,4 +801,6 @@ CONF_Int16(jdbc_connection_pool_size, "8");
 // The default value is set as the THREAD_POOL_SIZE of RoutineLoadTaskScheduler of FE.
 CONF_Int32(internal_service_async_thread_num, "10");
 
+// probability_of_inject% to inject error.
+CONF_Int32(probability_of_inject, "10000");
```

Comment:
what's the valid range for `probability_of_inject`? please comment it.

Reply:
Added